pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go

```diff
@@ -138,7 +138,7 @@ vet: ## Run go vet against code.
 
 .PHONY: test
 test: manifests generate fmt vet setup-envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -race -coverprofile cover.out
 
```

Comment:
Also, see that the changes here broke the tests

```
      #15 [builder 9/9] RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -race -a -o manager cmd/main.go
      #15 0.430 go: -race requires cgo; enable cgo by setting CGO_ENABLED=1
      #15 ERROR: process "/bin/sh -c CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -race -a -o manager cmd/main.go" did not complete successfully: exit code: 2
      ------
```

It does not seems to work.
So, can you please check out?